hypertools/plot/plot.py

```diff
@@ -60,9 +59,9 @@ def plot(x, fmt=None, marker=None, markers=None, linestyle=None,
         A list of labels for each point. Must be dimensionality of data (x).
         If no label is wanted for a particular point, input None.
 
-    legend : list or bool
```

Comment:
ah, yes - we merged this PR after you forked..we will want to `rebase` your code wrt hypertool's master. I think the right way to do this is `git rebase origin/master`, if you want to give it a try. You may need to set up a remote to do this (`git remote add origin https://github.com/ContextLab/hypertools`) What happens is that 1) your changes are undone, 2) your fork is updated to the latest code on hypertools master and 3) your changes are added back on top of the latest code.  If all goes smoothly, the legend code will be updated.  If there are merge conflicts, you would need to fix them, commit them and then continue the rebase.  If you want to try this, you can always `git rebase --abort` if things get messed up.